app/javascript/app.vue

```diff
@@ -45,13 +50,17 @@ export default {
   },
   computed: {
     ...mapGetters({
-      'products': 'onSaleProducts',
+      'products' : 'groupByCategory',
     }),
     ...mapState([
       'status',
     ]),
-    sortedProductList() {
-      return [...this.products].sort((a, b) => a.name.localeCompare(b.name));
+    productsArray() {
+      return [
+        {title: 'Snacks', products: this.products('snacks')},
+        {title: 'Bebestibles', products: this.products('drinks')},
```

Comment:
A space is required after '{'                object-curly-spacing<br>A space is required before '}'               object-curly-spacing